assets/section-featured-blog.css

```diff
@@ -1,12 +1,15 @@
+.blog {
+  padding-bottom: 3rem;
+}
+
```

Comment:
You are right but in my opinion we should not add an extra padding because eventually users see a page that was saved and if we add an extra padding to align the way things look in the editor that way we increase a final look of the page. Does it make sense? 🤔